src/components/Torrent/DashboardItems/Tracker.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <v-flex v-if="torrent.tracker" xs6 sm1 md1>
+    <div class="caption grey--text">
+      {{ $t('tracker') }}
+    </div>
+    <v-chip small class="trackers white--text caption">
+      {{ trackerString }}
+    </v-chip>
+  </v-flex>
+</template>
+
+<script lang="ts">
+import { TorrentDashboardItem } from '@/mixins'
+import { defineComponent } from 'vue'
+import { Torrent } from '@/models'
+
+export default defineComponent({
+  name: 'Tracker',
+  mixins: [TorrentDashboardItem],
+  props: {
+    torrent: Torrent
+  },
+  computed: {
+    trackerString() {
+      let trackerUrl = this.torrent.tracker.match(/:\/\/([^\/]+\.)?([^\/\.]+)\.[^\/\.:]+/i)
```

Comment:
Can we extract this matching, that's also used in the other function to a helper or something?

Perhaps write some unit tests for it to guarantee its functionality. 🤔

Reply:
I'm not familiar with nodejs, maybe the second commit will be better ;)